roles/openshift_clock_facts/tasks/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Set clock facts
+  openshift_facts:
+    role: "{{ item.role }}"
+    local_facts: "{{ item.local_facts }}"
+  with_items:
+  - role: clock
+    local_facts:
+      enabled: "{{ openshift_clock_enabled | default(False) }}"
```

Comment:
Also, it would probably be better to use default(None) here and set the default for openshift.clock.enabled to False in openshift_facts.